src/components/EntityInfo.vue

```diff
@@ -180,6 +195,29 @@ export default class EntityInfo extends Mixins(EnumMixin) {
     // assume Business Profile URL is always reachable
     window.location.assign(businessProfileUrl)
   }
+
+  /** Track route for breadcrumb. */
+  @Watch('$route', { immediate: true })
+  private getBreadCrumbs (): Array<BreadcrumbInterface> {
+    const breadCrumbs = this.$route?.meta?.breadcrumb
+    // The Current Account data only available in deployed environments
+    const currentAccount = JSON.parse(sessionStorage.getItem('CURRENT_ACCOUNT'))
+
+    return [
+      {
+        text: 'My Account',
+        disabled: false,
+        href: `${sessionStorage.getItem('AUTH_URL')}account/${currentAccount?.id}/business`
```

Comment:
There's actually a short-cut URL to this page, eg.
https://dev.bcregistry.ca/cooperatives/auth/business

^^ auth-web auto-forwards to the correct account id based on whatever's in the session storage (I think)

Reply:
Hmm, ok fair enough. I wasn't sure as locally i didn't have anything for Current Account, wasn't able to test that. I'll try this out.

Reply:
Yeah but you already assume you have current account in session storage -- isn't that the data auth-web needs for the shortcut?

Reply:
Perhaps, i think so. But i couldn't guarantee it goes to the specific manage business dashboard testing from local, so i tried to be explicit. I can remove it and we'll assume it does what we're hoping?

Reply:
Edit* It works!

Reply:
Does it work if you start from the Create UI (instead of starting from auth-web)? Wondering if the necessary data is set in this scenario, and if not whether it handles it anyway.

Reply:
Sorry, i should have clarified. You were correct, it handles it already by using the short cut URL.